### ISSUE TITLE
[Bug]: MSI installer not setting file associations

### ISSUE THREAD

**Issue by georgeescott** [sqlitebrowser/sqlitebrowser#3006]
### What did you do?

Installed the latest 3.12.2 MSI installer

### What did you expect to see?

Expected that computer file associations (.db, .db3, .sqlite and .sqlite3) to be set during install

### What did you see instead?

Nothing in the registry set for file associations, nothing in HKCR.

### DB4S Version

3.12.2

### What OS are you seeing the problem on?

Windows

### OS version

Windows 10 20H2

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by chrisjlocke** [comment]
No, I don't believe the installer has ever set the associations.  In Windows, you can right-click a file, select Properties, click 'Change' and select DB4S.  The trouble with SQLite is that as there is no 'standard' extension (some people use .db, some .sqlite, etc) we could set a range, but someone will also ask for another one to add, or will get upset that their associations have been broken.
Guess we could add another option in the installer process... 
I'll tag this as an enhancement and will look at the installer - it's something i wasnted to look into anyway as I'm not overly familiar with Wix, which is the installer package we use.
